src/com/sidebar/guides/guides.js

```diff
@@ -0,0 +1,31 @@
+import {h, Component}	from 'preact/preact';
+import SVGIcon			from 'com/svg-icon/icon';
+
+export default class SidebarGuides extends Component {
+	constructor() {
+	}
+
+	render(props, state) {
+		var Links = {
+			"Rules": {"icon": "article", "href": "/events/ludum-dare/rules"},
+			"Game Hosting Guide": {"href": "/events/ludum-dare/hosting-guide"}
+		};
+
+		return (
+			<div class="sidebar-base sidebar-shortlist sidebar-guides">
+				<div class="-title _font2">
+					<SVGIcon baseline>question</SVGIcon> <span class="-text">Guides</span>
+				</div>
+				{Object.keys(Links).map(key => {
+					const icon = Links[key].icon != null ? <SVGIcon baseline>{Links[key].icon}</SVGIcon> : null;
```

Comment:
If you add `gap` to the SVGIcon tag you can skip the space injection further down